### PR TITLE
Data transfer to s3

### PR DESCRIPTION
**Pull request by NilsSturma**
Local tests succesful. Next week want to test it on S3

### CODE REVIEW

File: client/sender.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    if os.listdir("{0}/pictures".format(config.LOCAL_PATH_RECORDINGS)):
+
+        # Collect complete the whole data and store it in a list
+        data = wrap_recordings(config.LOCAL_PATH_RECORDINGS)
+
+        token = auth_client(config.MSG_BROKER_BASE_URL, config.USER, config.PASSWORD)
+        success = post_data(data, config.MSG_BROKER_BASE_URL, token)
+
+        if success:
+            remove_files(config.LOCAL_PATH_RECORDINGS, ["pictures", "distances", "outputs"])
```
Comment (ssalb) [inline client/sender.py]: are you removing all files here, right? is that intended? new records could be generated in the meantime, so probably that's not what you want :P

File: server/app/process_queue.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    """ Write a dataframe to a CSV on S3 """
+
+    csv_buffer = StringIO()
+    dataframe.to_csv(csv_buffer, sep=",", index=False, header=False)
+
+    s3_resource = boto3.resource("s3")
+    s3_resource.Object(config.S3_BUCKET, filename).put(Body=csv_buffer.getvalue())
+
+
+# TODO: Write this function
```
Comment (ssalb) [inline server/app/process_queue.py]: is this a real TODO? or is it already done?

File: server/app/process_queue.py [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+                # timestamp
+                index.append(q['ts'])
+
+                # picture
+                pic_binary = q['pic'].encode("utf-8")
+                img = base64.b64decode(pic_binary)
+
+                # send it to S3
+                try:
+                    pass
```
Comment (ssalb) [inline server/app/process_queue.py]: is this intended? :D

**Review by ssalb — COMMENTED** [review]
Thanks @NilsSturma!

maybe you can move the "recordings" directory inside the client, as it's part of it

File: client/sender.py (hunk 1 above)
Comment (NilsSturma) [reply client/sender.py]: Ohh that's a good point! It was intended but in a different way, I changed it

File: server/app/process_queue.py (hunk 2 above)
Comment (NilsSturma) [reply server/app/process_queue.py]: This was done, sorry

File: server/app/process_queue.py (hunk 3 above)
Comment (NilsSturma) [reply server/app/process_queue.py]: I had this for testing reasons, removed it

**Review by ssalb — APPROVED** [review]
Great! Thanks @NilsSturma

**Pull request merged**